### ISSUE TITLE
Error when swapping "thick-green-border" and "smaller-image" within the class field

### ISSUE THREAD

**Issue by harjaz** [freeCodeCamp/freeCodeCamp#12813]
Challenge [Add Borders Around your Elements](https://www.freecodecamp.com/challenges/add-borders-around-your-elements) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

Swapping "thick-green-border" and "smaller-image" within the class field for the image doesn't pass the challenge despite having an identical result. The code below was accepted after I switched them

My code:
```html

<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  .red-text {
    color: red;
  }

  h2 {
    font-family: Lobster, Monospace;
  }

  p {
    font-size: 16px;
    font-family: Monospace;
  }

  .smaller-image {
    width: 100px;
  }
  
  .thick-green-border{
    border-width:10px;
    border-style:solid;
    border-color:green;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<img class="thick-green-border smaller-image" src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back. ">

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>

```

**Comment by dhcodes** [comment]
@harjaz Do you mean the order of the classes? Ex. `thick-green-border smaller-image` versus `smaller-image thick-green-border`? 

I ask because I tried them both ways and it passed for me in Chrome. Blocking for now as unable to replicate.

**Comment by celineguild** [comment]
I am getting the same issue.

Ordering it as: thick-green-border smaller-image - passes the border related requirements and fails the latter
Ordering it as: smaller-image thick-green-border - passes the image related requirements and fails the latter

**Comment by raisedadead** [comment]
Please clarify more as requested.

**Comment by mchl326** [comment]
I had to switch thick green border and smaller image as well in order for the code to work harjaz. Not sure why this was the case.

**Comment by mobieonekinobi** [comment]
Page is unresponsive to code changes for smaller-image and thick-green-border. I have tried to move things around and even remove a section and it does not respond to those changes. Tried different browsers too. Adding smaller-image and thick-green-border both have issues. My code is correct because it works running it on other editors but not working in CFC editor. I think we want to know if there is a workaround for our side and/or if CFC is working on a fix? Thanks!
